src/app/blog/(posts)/running-and-scaling-dedicated-game-servers-in-godot/page.mdx

```diff
@@ -0,0 +1,63 @@
+import image from './opengraph-image.png';
+
+export const config = {
+  author: 'nathan-flurry',
+  date: '2024-05-09',
+  title: `Everything You Need to Know About Running & Scaling Dedicated Game Servers in Godot`,
+  description: `Learn the key aspects of running and scaling dedicated game servers for your Godot-based multiplayer game`,
+  category: 'guide',
+  keywords: [
+    'godot',
+    'multiplayer',
+    'dedicated-game-servers',
+    'server-scaling',
+    'networking-protocols',
+    'multiplayer-synchronization',
+    'testing'
+  ],
+  images: {
+    hero: {
+      image,
+      alt: 'hero'
+    }
+  }
+};
+
+## Introduction
+
+When it comes to creating a shared, multiplayer game world in Godot, setting up and managing dedicated game servers is a crucial aspect of the development process. In this blog post, we'll explore the key considerations and best practices for running and scaling dedicated game servers for your Godot-based multiplayer project.
+
+## Choosing the Right Networking Protocol
```

Comment:
On my first pass, this being the first important paragraph seems out of place. I'd expect something about Godot first I think. Will circle back once I finish.